extraction/src/test/scala/org/broadinstitute/monster/encode/extraction/EncodeExtractionPipelineBuilderSpec.scala

```diff
@@ -0,0 +1,264 @@
+package org.broadinstitute.monster.encode.extraction
+
+import better.files.File
+import org.broadinstitute.monster.common.PipelineBuilderSpec
+import org.broadinstitute.monster.common.msg.MsgOps
+import upack._
+
+import scala.collection.mutable
+
+object EncodeExtractionPipelineBuilderSpec {
+  val fakeIds = 1 to 1
+
+  // biosamples
+  val biosampleParams = ("organism.name", "human")
+
+  val biosampleOut = Obj(
+    Str("@graph") -> new Arr(
+      fakeIds.map { i =>
+        Obj(
+          Str("@id") -> Str(i.toString),
+          Str("donor") -> Str(i.toString),
+          Str("accession") -> Str(i.toString)
+        ): Msg
+      }.to[mutable.ArrayBuffer]
+    )
+  )
+
+  // donors
+  val donorParams = ("@id", "1")
+
+  val donorOut = Obj(
+    Str("@graph") -> new Arr(
+      fakeIds.map { i =>
+        Obj(
+          Str("@id") -> Str(i.toString)
+        ): Msg
+      }.to[mutable.ArrayBuffer]
+    )
+  )
+
+  // libraries
+  val libraryParams = ("biosample.accession", "1")
+
+  val libraryOut = Obj(
+    Str("@graph") -> new Arr(
+      fakeIds.map { i =>
+        Obj(
+          Str("@id") -> Str(i.toString),
+          Str("accession") -> Str(i.toString)
+        ): Msg
+      }.to[mutable.ArrayBuffer]
+    )
+  )
+
+  // replicates
+  val replicateParams = ("library.accession", "1")
+
+  val replicateIds = 1 to 2
```

Comment:
Should there be a downstream step that includes multiple expected queries because of this, or are we not chaining any objects off of replicates?

Reply:
If nothing's being chained from replicates, can we bump one of the other objects to return multiple things, to be sure that chaining works properly?

Reply:
experiments are chained from replicates; I have 2 here so that we get at least one thing for `experiment` and one for `fcexperiment